cpu/msp430_common/Kconfig

```diff
@@ -0,0 +1,9 @@
+# Copyright (c) 2020 Freie Universitaet Berlin
```

Comment:
This is a common CPU file. You will need to include this from Kconfig files in the CPUs that use this common:
- cc430
- msp430fxyz